### PR TITLE
Feature/user address

### PR DESCRIPTION
**Pull request by lbebber**
#### What is the purpose of this pull request?
Adds `UserAddress` component, intended for inclusion on the page's header.

screen shot 2019-02-25 at 18 00 58

screen shot 2019-02-25 at 18 01 10

### CODE REVIEW

File: react/components/UserAddress/ChangeAddressIcon.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const ChangeAddressIcon = () => {
+  return (
+    <svg width="21" height="27" viewBox="0 0 21 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M19.9655 10.4828C19.9655 16.3448 10.4828 26 10.4828 26C10.4828 26 1 16.431 1 10.4828C1 4.44828 5.91379 1 10.4828 1C15.0517 1 19.9655 4.53448 19.9655 10.4828Z" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (lucasecdb) [inline react/components/UserAddress/ChangeAddressIcon.js]: shouldn't `stroke-width`, `stroke-miterlimit`, `stroke-linecap` and `stroke-linejoin` be capitalized like `strokeWidth`?

File: react/components/UserAddress/ChangeAddressIcon.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      <path d="M10.4827 13.069C11.911 13.069 13.0689 11.9111 13.0689 10.4828C13.0689 9.05443 11.911 7.89655 10.4827 7.89655C9.05437 7.89655 7.89648 9.05443 7.89648 10.4828C7.89648 11.9111 9.05437 13.069 10.4827 13.069Z" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (lucasecdb) [inline react/components/UserAddress/ChangeAddressIcon.js]: same as above

File: react/components/UserAddress/index.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
+import React  from 'react'
+import PropTypes from 'prop-types'
+import { orderFormConsumer, contextPropTypes } from 'vtex.store-resources/OrderFormContext'
+import ChangeAddressIcon from './ChangeAddressIcon'
+
+import Container from '../Container'
+
+import { FormattedMessage } from 'react-intl'
```
Comment (lucasecdb) [inline react/components/UserAddress/index.js]: place this import before the relative ones

File: react/components/UserAddress/ChangeAddressIcon.js (hunk 1 above)
Comment (lbebber) [reply react/components/UserAddress/ChangeAddressIcon.js]: It works like this, so 🤷‍♂️

Comment (lucasecdb) [reply react/components/UserAddress/ChangeAddressIcon.js]: I think react warns if the props are like that, but I guess it's fine

**Pull request merged**